@types/index.d.ts

```diff
@@ -128,17 +128,19 @@ declare module "@vonage/client-sdk-video-react-native" {
 
   interface OTSessionProps extends ViewProps {
     /**
-     * TokBox API Key
+     * Your Vonage application ID. (When using the Vonage Video React Native SDK
+     * with OpenTok projects, you pass in an OpenTok API key. But for Vonage
+     * applications, pass in the application ID.)
```

Comment:
Should it be Application ID?

Reply:
I prefer not to capitalize things that are not inherently proper nouns. I don't know what the Vonage style is.